### PR TITLE
Updates to system_module_config.json

### PR DESCRIPTION
**Pull request by InBrewJ**
@pisuke I may have messed up the part about the 'Required Info' for bacnet.version...

### CODE REVIEW

File: qualification/system_module_config.json [hunk 1]
```diff
@@ -1,29 +1,92 @@
...
-    "switch": {
+    "mudgee": {
       "enabled": true
     },
-    "brute": {
+    "nmap": {
       "enabled": true
     },
-    "bacext": {
+    "switch": {
```
Comment (pisuke) [inline qualification/system_module_config.json:21]: @InBrewJ Do we need to add the PoE stuff in here too?

**Review by pisuke — APPROVED** [review]
@InBrewJ Looks OK to me. 
I'm approving, but can you add the PoE configuration variables to the JSON file too?
Thanks!

File: qualification/system_module_config.json (hunk 1 above)
Comment (InBrewJ) [reply qualification/system_module_config.json:21]: @pisuke Done. Do we need to add something about poe tests into the required Pass/fail section too?

Comment (pisuke) [reply qualification/system_module_config.json:21]: Yup, good catch, please proceed adding this too, thanks!

Comment (pisuke) [reply qualification/system_module_config.json:21]: OK, I've added all the configuration bits for the existing tests.

**Pull request merged**